specs/merge/validator.md

```diff
@@ -29,7 +32,7 @@ This document represents the changes to be made in the code of an "honest valida
 
 ## Prerequisites
 
-This document is an extension of the [Altair -- Honest Validator](../altair/validator.md) guide.
+This  documens is an extension of the [Altair -- Honest Validator](../altair/validator.md) guide.
```

Comment:
accidentally changed?

Reply:
I'm surprised that `codespell` allowed it. 🤔